grafana-plugin/tsconfig.json

```diff
@@ -6,8 +6,8 @@
     "rootDirs": ["src", "frontend_enterprise/src"],
     "baseUrl": "src",
     "typeRoots": ["./node_modules/@types"],
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
+    "noUnusedLocals": false,
+    "noUnusedParameters": false,
```

Comment:
why so?

Reply:
but actually compilation doesn't fail, it just show ts errors but doesn't stop the rebuilding process, at least for me...

Reply:
But when you stop the process and your try again, then it fails for me